cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -2607,29 +2657,11 @@ Slice::Python::dynamicCompile(const vector<string>& files, const vector<string>&
         generatedModules.push_back(fragment.moduleName);
     }
 
-    // The list of non generated modules that are imported by the generated modules.
-    vector<string> builtinModules{
-        "Ice.FormatType",
-        "Ice.Object",
-        "Ice.ObjectPrx",
-        "Ice.ObjectPrx_forward",
-        "Ice.OperationMode",
-        "Ice.StringUtil",
-        "Ice.UserException",
-        "Ice.Value",
-        "Ice.Value_forward",
-        "abc",
-        "dataclasses",
-        "enum",
-        "typing"};
```

Comment:
This list is not required, we only check dependencies that are part of the current compilation and builtin modules are by definition not part of the Slice compilation.